ProtocolHandler.pm

```diff
@@ -10,6 +10,22 @@ use Slim::Utils::Log;
 use Slim::Utils::Misc;
 use Slim::Utils::Prefs;
 
+use List::Util qw(min max first);
+use Slim::Utils::Errno;
+
+# Size of the _sysread chunksize AND buffers, NOT to be confuesed with same applied to pipeline and/or player.
+
+use constant MAX_OUT    => 512*1024*1024; # safety limit, the out should always be empty, if reached _sysread stops reading from source.
+
+use constant MIN_OUT    => 256*1024;      # data are not transferred to output until this limit is reached in input.
+use constant RANGE_SIZE => 1024*1024;     # size of range in the http get request, not really the chunksize, but similar.
+
+# Note that playback will not start untill the firs http get returns, so the greather value between RANGE_SIZE and MIN_OUT, is the real 
+# threshold.
+
+# Seconds of delay before playback starts (initial buffering seconds)
+use constant BUFFERING_SECONDS  => 5;
```

Comment:
Why not just use bufferSecs? This is redundant.

Reply:
Becouse buffer secs is used with different meaning in play and in rebuffering.